spring-cloud-deployer-spi/src/main/java/org/springframework/cloud/deployer/spi/app/AbstractActuatorTemplate.java

```diff
@@ -136,6 +177,16 @@ else if (count > 1) {
 				.findFirst();
 	}
 
-	protected abstract String actuatorUrlForInstance(AppInstanceStatus appInstanceStatus);
+	private Optional<String> prepareDefaultAthentication(AppAdmin appAdmin) {
+		Optional<String> encodeBasicAuth;
```

Comment:
**[TinyNit]** the declaration and assignment can be combined into single statement.

Reply:
Fix it in the merge.